### PR TITLE
DUOS-1590[risk=no] ux-merge-researcher-review-edit-user-page

### PR DESCRIPTION
**Pull request by shaemarks**
[Link to Jira ticket](https://broadworkbench.atlassian.net/browse/DUOS-1590)
Brief summary of changes: 

- add information from Researcher Review page to Edit User modal on the Admin Manage Users page 
----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/ResearcherReview.js [hunk 1]
```diff
@@ -47,7 +46,7 @@ class ResearcherReview extends Component {
 
   async findResearcherInfo() {
 
-    const user = await User.getById(this.props.match.params.dacUserId);
+    const user = await User.getById(this.props.userId);
```
Comment (rushtong) [inline src/pages/ResearcherReview.js:49]: I think this change breaks the "Review Researcher" button on the "Manage Users" page. I think that button is no longer necessary. Can we remove it for this PR? That would solve the problem that it doesn't work 😄

**Review by rushtong — CHANGES_REQUESTED** [review]
Code looks good, works for what we want it to do. See comment for my one suggestion before this is ready.

File: src/pages/ResearcherReview.js (hunk 1 above)
Comment (shaemarks) [reply src/pages/ResearcherReview.js:49]: I removed the Researcher Review column. There's currently some extra space on the right from where it used to be, but I'm not sure if I should look into that or leave it for now.

Comment (rushtong) [reply src/pages/ResearcherReview.js:49]: You can leave that in place, not necessary to shrink it down.

**Review by rushtong — APPROVED** [review]
Changes look good, thank you! 👍🏽

**Pull request merged**